packages/warning/src/utils.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Object map tracking messages which have been logged, for use in ensuring a
+ * message is only logged once.
+ *
+ * @type {Record<string, true | undefined>}
+ */
+export const logged = Object.create( null );
```

Comment:
Is it `Object.create( null )` instead of just `{}` to avoid prototype pollution? Very clever! 👍

Reply:
Yes, I copied it over from `@wordpress/deprecated`.

Reply:
Would we be better off with `Set`?

Reply:
Good point, we could use `Set` 👍🏻

Reply:
https://github.com/WordPress/gutenberg/pull/33380 should address it.